### ISSUE TITLE
Collection details page - single page view

### ISSUE THREAD

**Issue by anarchivist** [sul-dlss/arclight#150]
There should be a button that allows you to view a "single page view" of an entire collection description, including the overview, and then a full, hierarchical representation of the inventory.

**Comment by mejackreed** [comment]
During standup on 5/19 we discussed this ticket in the context of technical things that need to happen before this can be approached. During the discussion we noticed ArchiveSpace's feature to export out as a PDF. For MVP simplification we were thinking that this functionality could be accomplished in the following way:

 - Rather than try and create this view in ArcLight (hard) lets just add download links to these objects.
 - adopters can link the ead or pdf view of the ead from within the ead xml
 - these then can show up to be downloaded

@ggeisler mentioned needing to change the label

**Comment by anarchivist** [comment]
Could you explain more? I'm not sure that the ticket is intended to address the EAD or PDF export as written. (I realize now I didn't include it in out of scope, but it should be understood as such).

**Comment by ggeisler** [comment]
@anarchivist I believe @mejackreed feels that generating an HTML single-page view of the collection (with the complete inventory expanded) is technically difficult. I mentioned that while offering single-page HTML, PDF, and EAD choices would be ideal, if the single-page HTML is currently a big hurdle we should at least try to offer the PDF version (I believe I remember from our user research that researchers often like to keep and use a PDF of the collection they are working with, so this seems worth pursuing to some level of effort, though I'm not making any argument for prioritization, other than I think it is moderately important).

What we discussed during standup was how we might provide the PDF option without putting the responsibility for the export on ArcLight. In other words, the implementing institution would be responsible for putting collection PDFs (and EADs) in say, the `/data` directory, and if ArcLight found a `12345.pdf` in the `/data` directory of collection `12345` then we would show the Single Page links on the collection detail page with link(s) to the appropriate file(s).

Not every collection would necessary have generated a PDF, but if one doesn't exist, we just wouldn't show a link for it. Since AS does have a PDF export feature, though, it seems reasonable to expect that many collections could produce PDFs relatively easily.

**Comment by anarchivist** [comment]
I think offering the PDF version generally feels like a stretch for the MVP - I don't know that it adds much value at this point, especially 1) if we're using generated or dummy PDFs and 2) we're not relying on a live ArchivesSpace endpoint. Thoughts?

**Comment by ggeisler** [comment]
@anarchivist: I don't see it as us providing PDFs as much as designing a feature that will provide links to PDFs if they exist. We'd just have to write some code for the Collection show page that looks at a designated directory to see if there is a PDF named with the ID of the current collection and if so, show the Single Page button (maybe renamed, depending on exactly what we do here) with a link to that PDF.

From our discussion at standup today, that seemed like a doable and relatively low-effort approach, but if we pursued this it would be good to get your and @jkeck's thoughts and evaluate it more thoroughly.  

That said, I don't feel strongly that this feature needs to be implemented as part of the MVP. I'm happy for you to make that decision, @anarchivist.

**Comment by anarchivist** [comment]
Hm. I'll give it some thought - but perhaps the EAD/PDF part should be broken out in a different ticket.

**Comment by ggeisler** [comment]
Yes, the EAD/PDF part is completely different work from making a single-page HTML version, so it definitely makes sense to have separate tickets for them.

**Comment by gwiedeman** [comment]
Possibly duplicative of #37
